### ISSUE TITLE
RTL Support

### ISSUE THREAD

**Issue by alash3al** [orchidsoftware/platform#1397]
Sometimes I want to use orchid for RTL based apps, but it needs some effort to do that, what I do to build and rtl app?

- create a `dir` attribute of the html tag with `rtl` value
- when some content fails to be rtl compatible I use `rtlcss` tool

but there are still some other glitches, 

I know that rtl isn't your top priority, but at least what is the best option to do that from my side?

**Comment by tabuna** [comment]
I may not correctly estimate the result, but this is how the RTL version looks like now with the correct indication of the attribute:

![image](https://user-images.githubusercontent.com/5102591/102338835-6a451c80-3fa5-11eb-86e3-a40d9c7018ad.png)

The most obvious problem I see is incorrect indentation. But we have an issue with updating bootstrap #1152
As far as I understand, switching to it should partially solve this problem, since they changed the name of the styles for margin and padding:

> Renamed .left-* and .right-* to .start-* and .end-*.

They also use the tool that you do: https://getbootstrap.com/docs/5.0/getting-started/rtl/
> when some content fails to be rtl compatible I use rtlcss tool



We need to make the transition to bootstrap 5.
And also write a simple function that, depending on the language used, returns LTR or RTL

```php
<HTML dir="{{ Orchid\Platform\Support\Locale::currentDir() }}">
```

**Comment by tabuna** [comment]
I started working on porting my codebase to bootstrap 5 in the master branch. 
I would be glad if you work on adding this feature.

**Comment by astersnake** [comment]
Resolved in #2155.
